### ISSUE TITLE
meow not activated in help mode (`C-h t`)

### ISSUE THREAD

**Issue by zenny** [DogLooksGood/meow#6]
Hi,

I have updated the meow config with:

```
(meow-leader-define-key
 '("k" . kill-buffer)
 '("l" . goto-line)
 '("h" . other-window)
 '("o" . delete-other-windows)
 '("-" . split-window-below)
 '("/" . swiper)
 '("\\" . split-window-right)
 '("m" . magit-status)
 '("f" . find-file)
 '("F" . find-file-literally))

(meow-leader-define-mode-key
 'emacs-lisp-mode
 '("RET" . eval-buffer)
 '("SPC" . eval-defun))
```
but when in the emacs-tutorial, meow mode appears to be deactivated. 

HOW TO REPRODUCE?

To reproduce, just `C-h t` and check the mode, you would only find a squared box in the modeline (in scimax), in my case).

**Comment by DogLooksGood** [comment]
The tutorial is in fundamental mode, I haven't find a good way to automatically enable, for example, normal state in fundamental mode.

I think there should be a solution for this.

**Comment by KiteAB** [comment]
Is this issue have been solved?

**Comment by DogLooksGood** [comment]
Not yet, but Meow will bind `esc` to switch to NORMAL state globally, so in tutorial, you can use `esc` back to normal state.

**Comment by DogLooksGood** [comment]
fixed in 103151175a9a3e3930d38e169846eca933683b52
